Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -2674,8 +2674,6 @@ private class ClassWithNonBrowsableMembers
 
         private class ClassWhereMemberThatCouldBeNonBrowsableIsBrowsable
         {
-            public int BrowsableField = -1;
```

Comment:
Hmm.. no test failed 🤔

Reply:
Theoretically a test could pass for the wrong reason or mutation score could have decreased.
I'd like know what something was supposed to do before deleting it.

@logiclrd since you implemented this in #1827 you might have some insights?